.changeset/empty-tips-thank.md

```diff
@@ -0,0 +1,8 @@
+---
+"@salt-ds/lab": patch
+---
+
+- Change `NavigationItem` flex alignment so it works well with longer labels
+- Add new `--salt-size-indicator` to `--navigationItem-bar-size` css var
```

Comment:
don't think we need to mention the css changes. Or do we?

Reply:
probably not, I'll remove this